server/test/BraveAlerterConfiguratorTest.js

```diff
@@ -1,280 +1,280 @@
+/* eslint-disable import/no-extraneous-dependencies */
```

Comment:
What were the extraneous dependencies that it didn't like?

Reply:
It wants mocha and chai under dev dependencies instead of project

https://bravely.slack.com/archives/CE32NB85N/p1611562529027800?thread_ts=1611547219.027500&cid=CE32NB85N

Reply:
I agree. It should be under dev dependencies instead of project.

Reply:
This is interesting. It looks like this shouldn't be a problem since 2016: https://github.com/airbnb/javascript/commit/2f9dddbe205145512978a9f07b08fe987505d4d7

Reply:
Hmmmm I can look into that - and yeah I am pretty sure they are (I looked last night and meant to answer this) so maybe if I take this away it will work now??? I will check all these things out and have better answers before you're back on Thursday :)

Reply:
Okay I had it backwards, it wants them in regular dependencies over dev 🤔 

Also this test passes still when reverted to what's in `master` with all the other things still the updated ones

Reply:
@trisapeace I'm coming up on being done merging in the changes from master and wanted to check on this as it comes up in the one new test file as well, should I just move these to regular dependencies, or leave it like this with the disable??

Reply:
No, they should not be moved to regular dependencies. 

Could you please try to put the disable into the ESLint config file instead of as a comment in the test file? You should be able to disable it just for files in the /test folder. Then it won't be a problem for new test files that are created later.

Reply:
Yup can do, I already have something in there for tests specifically & will be moving any modifications to the central one once we’ve solidified what we want :)